src/main/frontend/format/block.cljs

```diff
@@ -257,6 +257,26 @@
               body)]
     (assoc block :body body)))
 
+(defn extract-filetags
```

Comment:
happy to add a test once we agree on the code, would https://github.com/logseq/logseq/blob/master/src/test/frontend/db/model_test.cljs be the right place?

Reply:
hmm, `model_test.cljs` is for testing the database related functions. Currently, the code to extract the page properties is located at https://github.com/logseq/logseq/blob/master/src/main/frontend/format/mldoc.cljs#L80